Example/src/screens/Events.tsx

```diff
@@ -256,14 +287,10 @@ const App = (): JSX.Element => {
 
   return (
     <ToastProvider>
-      <Stack.Navigator
-        screenOptions={{
-          direction: I18nManager.isRTL ? 'rtl' : 'ltr',
-          nativeBackButtonDismissalEnabled: true,
-        }}>
+      <Stack.Navigator>
```

Comment:
Is direction not supported on v6?

Reply:
Yep - currently it is fully automated on react-navigation side!